packages/web/src/components/spa/HackAUBG/GradingCriteria/GradingCriteriaDesktop/grading_criteria_desktop.css

```diff
@@ -0,0 +1,48 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:wght@200,300,400,500,600,700,900&display=swap');
```

Comment:
Is this the only place where you import Roboto? If that's the case, that's good, otherwise, please import it only in the root css where we import Poppins.